sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -358,8 +358,45 @@ The azure-core [reference documentation](https://azuresdkdocs.blob.core.windows.
 describes available configurations for retries, logging, transport protocols, and more.
 
 ## Troubleshooting
+
+### General
 The Text Analytics client will raise exceptions defined in [Azure Core](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/core/azure-core/README.md).
 
+### Logging
+This library uses the standard
+[logging](https://docs.python.org/3.5/library/logging.html) library for logging.
```

Comment:
nit: Would it help to maintain the hyperlinks in this file if they are summarized at the end of the file.